.github/workflows/power9_tpls.yml

```diff
@@ -0,0 +1,106 @@
+    
+    steps:
+      - name: checkout_kokkos_kernels
+        uses: actions/checkout@v3
+        with:
+          path: kokkos-kernels
+
+      - name: checkout_kokkos
+        uses: actions/checkout@v3
+        with:
+          repository: kokkos/kokkos
+          ref: ${{ github.base_ref }}
+          path: kokkos
+
+      - name: configure_kokkos
+        run: |
+          mkdir -p kokkos/{build,install}
+          cd kokkos/build
+          GPP=$(which g++)
+          cmake -DCMAKE_CXX_COMPILER=$GPP \
+            -DCMAKE_CXX_FLAGS="-O3 -Wall -Wunused-parameter -Wshadow -pedantic -Werror -Wsign-compare -Wtype-limits -Wignored-qualifiers -Wempty-body -Wclobbered -Wuninitialized" \
+            -DCMAKE_EXE_LINKER_FLAGS= \
+            -DCMAKE_INSTALL_PREFIX=$PWD/../install \
+            -DKokkos_ENABLE_OPENMP=ON \
+            -DKokkos_ARCH_VOLTA70=ON \
+            -DKokkos_ARCH_POWER9=ON \
+            -DKokkos_ENABLE_TESTS=OFF \
+            -DKokkos_ENABLE_EXAMPLES=OFF \
+            -DCMAKE_VERBOSE_MAKEFILE=ON \
+            -DCMAKE_CXX_EXTENSIONS=OFF \
+            -DCMAKE_CXX_STANDARD=17 \
+            -DBUILD_SHARED_LIBS=OFF \
+            -DKokkos_ENABLE_DEPRECATION_WARNINGS=OFF \
+            -DKokkos_ENABLE_DEPRECATED_CODE_4=OFF \
+            ..
+
+      - name: build_and_install_kokkos
+        working-directory: kokkos/build
+        run: make -j12 install
+
+      - name: configure_kokkos_kernels
+        run: |
+          mkdir -p kokkos-kernels/{build,install}
+          cd kokkos-kernels/build
+          GPP=$(which g++)
+          cmake \
+            -DCMAKE_CXX_COMPILER=$GPP \
+            -DKokkos_DIR=$PWD/../../kokkos/install/lib64/cmake/Kokkos \
+            -DCMAKE_CXX_FLAGS="-O3 -Wall -Wunused-parameter -Wshadow -pedantic -Werror -Wsign-compare -Wtype-limits -Wignored-qualifiers -Wempty-body -Wclobbered -Wuninitialized" \
+            -DCMAKE_INSTALL_PREFIX= \
+            -DKokkosKernels_ENABLE_TESTS_AND_PERFSUITE=OFF \
+            -DKokkosKernels_ENABLE_TESTS=ON \
+            -DKokkosKernels_ENABLE_PERFTESTS=ON \
+            -DKokkosKernels_ENABLE_EXAMPLES:BOOL=ON \
+            -DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=OFF \
+            -DKokkosKernels_INST_COMPLEX_DOUBLE=ON \
+            -DKokkosKernels_INST_DOUBLE=ON \
+            -DKokkosKernels_INST_ORDINAL_INT=ON \
+            -DKokkosKernels_INST_OFFSET_SIZE_T=ON \
+            -DKokkosKernels_INST_OFFSET_INT=ON \
+            -DKokkosKernels_INST_LAYOUTLEFT=ON \
+            -DKokkosKernels_ENABLE_TPL_ROCSPARSE=OFF \
+            -DKokkosKernels_ENABLE_TPL_ROCBLAS=OFF \
+            -DKokkosKernels_ENABLE_TPL_CUSOLVER=OFF \
+            -DKokkosKernels_ENABLE_TPL_BLAS=ON \
+            -DKokkosKernels_ENABLE_TPL_CUBLAS=OFF \
+            -DKokkosKernels_ENABLE_TPL_CUSPARSE=OFF \
+            -DCMAKE_EXE_LINKER_FLAGS="-lgfortran -lm" \
+            -DLAPACK_LIBRARIES=lapack \
+            -DBLAS_LIBRARIES=blas \
```

Comment:
That one might be openblas actually? Usually I also have to specify `-D BLAS_LIBRARY_DIR=/path/to/openblas/lib`
Same thing would be needed for LAPACK

Reply:
same as above WRT cm_test_all_sandia.

Reply:
Okay but that one is most likely not working at all. Do you have a config output for this command line?
Since the BLAS/LAPACK implementation used is supposedly openblas based on the name of the build, the library will not be able to find it.

Reply:
This has not been tested yet, only the 7 items checked off in the list above have been tested and confirmed to pass.

Reply:
Okay, in that case can we remove it from this PR?
My experience tells me that if we check it in knowing with issues and do not work them out immediately this will likely stay there forever or until something important goes wrong and we have to fix it will we are putting out a fire.